ProcessMaker/Http/Controllers/Api/UserTokenController.php

```diff
@@ -50,6 +50,41 @@ public function __construct(TokenRepository $tokenRepository, ValidationFactory
 
     /**
      * Display listing of access tokens for the specified user
+     *
+     * @OA\Get(
+     *     path="/users/{user_id}/tokens",
+     *     summary="Returns a specific user's tokens",
+     *     operationId="getTokens",
+     *     tags={"Personal Tokens"},
+     *     @OA\Parameter(
+     *         description="User id",
+     *         in="query",
```

Comment:
@AbdelilahLbardi  When testing the endpoints that `user_id` is being included as a query for example `/users/{user_id}/tokens?user_id=2` instead of `/users/2/tokens`. To fix this update `in="query"`  to be `in="path"`. 

Make sure to run `php artisan l5-swagger:generate` when making changes to update the `api-docs.json` file.

Reply:
✅  done!